specification/_types/Base.ts

```diff
@@ -73,7 +73,7 @@ export class ErrorResponseBase {
   // If the error is a string, it means that it was not caused by an exception on ES side, but on the HTTP routing layer.
   // This should never happen in clients, because we assume we will never send malformed request.
   error: ErrorCause
-  status: integer
+  status?: integer
```

Comment:
Status is always present in error responses, including in multisearch responses. See [ES code](https://github.com/elastic/elasticsearch/blob/8754c17a2d6218bce2041cddc572d2a58cef895f/server/src/main/java/org/elasticsearch/action/search/MultiSearchResponse.java#L161) and [the docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-multi-search.html#search-multi-search-api-response-body) "an object with error message and corresponding status code will be returned in place of the actual search response".

Keeping `error` and `status` required also plays an important role (in the Java client) to disambiguate some non-200 results where ES will return either a valid response or an error depending on the parameters (e.g. delete document that returns 404+error if the index doesn't exist but 404+valid response if the index exist but not the document).